src/System.Net.Http/src/uap/System/Net/HttpHandlerToFilter.cs

```diff
@@ -47,9 +50,27 @@ internal HttpHandlerToFilter(RTHttpBaseProtocolFilter filter)
             }
             cancel.ThrowIfCancellationRequested();
 
-
             RTHttpRequestMessage rtRequest = await ConvertRequestAsync(request).ConfigureAwait(false);
-            RTHttpResponseMessage rtResponse = await _next.SendRequestAsync(rtRequest).AsTask(cancel).ConfigureAwait(false);
+
+            RTHttpResponseMessage rtResponse;
+            try
+            {
+                rtResponse = await _next.SendRequestAsync(rtRequest).AsTask(cancel).ConfigureAwait(false);
+            }
+            catch (TaskCanceledException)
```

Comment:
I copied this pattern from here:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/src/System/Net/Http/DiagnosticsHandler.cs#L100

If it's wrong there, we can fix both in later PR.